cmd/admin-revoker/main.go

```diff
@@ -1,80 +1,14 @@
 package notmain
 
 import (
-	"bufio"
-	"context"
-	"crypto/x509"
-	"errors"
 	"flag"
 	"fmt"
-	"io"
 	"os"
-	"os/user"
-	"sort"
-	"strconv"
-	"sync"
-
-	"github.com/jmhodges/clock"
-	"google.golang.org/protobuf/types/known/timestamppb"
 
 	"github.com/letsencrypt/boulder/cmd"
-	"github.com/letsencrypt/boulder/core"
-	"github.com/letsencrypt/boulder/db"
-	berrors "github.com/letsencrypt/boulder/errors"
 	"github.com/letsencrypt/boulder/features"
-	bgrpc "github.com/letsencrypt/boulder/grpc"
-	blog "github.com/letsencrypt/boulder/log"
-	"github.com/letsencrypt/boulder/metrics"
-	"github.com/letsencrypt/boulder/privatekey"
-	rapb "github.com/letsencrypt/boulder/ra/proto"
-	"github.com/letsencrypt/boulder/revocation"
-	"github.com/letsencrypt/boulder/sa"
-	sapb "github.com/letsencrypt/boulder/sa/proto"
 )
 
-const usageString = `
```

Comment:
There's no longer a way to list available subcommands, but that's ok because this change is intended as a migration to the new admin tool and will help teams using old documented `admin-revoker` command.